osd_install_access_delete_cluster/creating-a-gcp-psc-enabled-private-cluster.adoc

```diff
@@ -1,21 +1,21 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="osd-creating-a-gcp-psc"]
-= Creating a GCP Private Service Connect enabled private cluster
+[id="creating-a-gcp-psc-enabled-private-cluster"]
+= Private Service Connect overview
 include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: osd-creating-a-gcp-psc-enabled-private-cluster
 
 toc::[]
 You can create a private {product-title} cluster on {GCP} using Google Cloud's security-enhanced networking feature Private Service Connect (PSC).
 
-include::modules/private-service-connect-overview.adoc[leveloffset=+1]
-
+include::modules/osd-understanding-private-service-connect.adoc[leveloffset=+1]
+include::modules/private-service-connect-psc-architecture.adoc[leveloffset=+1]
 include::modules/private-service-connect-prereqs.adoc[leveloffset=+1]
 
-include::modules/private-service-connect-create.adoc[leveloffset=+1]
-
-
-[id="additional-resources_{context}"]
+[id="additional-resources-psc_{context}"]
 == Additional resources
-For information on {product-title} on {GCP} cluster prerequisites, see xref:../osd_planning/gcp-ccs.adoc#ccs-gcp-customer-requirements_gcp-ccs[Customer Requirements].
+* For information about {product-title} on {GCP} cluster prerequisites, see xref:../osd_planning/gcp-ccs.adoc#ccs-gcp-customer-requirements_gcp-ccs[Customer Requirements].
+
+* For information about configuring your firewalls, see xref:../osd_planning/gcp-ccs.adoc#osd-gcp-psc-firewall-prerequisites_gcp-ccs[GCP firewall prerequisites].
 
-For information about configuring your firewalls , see xref:../osd_planning/gcp-ccs.adoc#osd-gcp-psc-firewall-prerequisites_gcp-ccs[GCP firewall prerequisites].
+* For information about creating an {product-title} on {GCP} using PSC and Workload Identity Federation, see
```

Comment:
Does this need to be say ..."using PSC with Workload Identity Federation..." maybe?